### DISCUSSION TITLE
Allow browser to pass HEVC support so if already supported it will play

### DISCUSSION THREAD

**Discussion by molitar** [pulse-browser/browser#274] [Ideas]
This is how Chrome handles it.  Chrome will play HEVC files IF the system already has support for it.  This way they don't have to pay for licensing but can provide the support.  It passes playing the hevc video to the hardware itself.

Right now Firefox is becoming useless as more people are using HEVC overseas and Firefox only plays them as audio files no video :( 

This is one of the reasons people are leaving Firefox video support sucks.

**Comment by trickypr** [comment]
If you want to contribute a patch, feel free to. I don't need it, so I wont implement it.
